### DISCUSSION TITLE
Trouble using a list as checkbox input in form

### DISCUSSION THREAD

**Discussion by ElTacitos** [johnlindquist/kit#1139] [Q&A] [answered]
I'm trying to use a list of string as value for checkbox in a form() but I all I can get is the last object I selected.
Here is what I currently have:
```javascript
const numbers = [1,2,3]
const formValues = await form(`
  numbers.map((num) => `
    <div>
      <input type="checkbox" id="${num}" name="selectedNums" value="${num}"\>
      <label for="${num}">${num}</label>
    </div>
  `).join("\n");
  <button type="submit">Submit</button>
`)
```

If I check 1 & 2 and then inspect `formValues` all I get is:
```json
{
    "selectedNums": "2"
}
```

When I run the same code but I wrote all the checkbox "one by one" I get the expected result (` "selectedNums": ["1","2"]`)

Am I doing something wrong ?

**Comment by johnlindquist** [comment, marked as answer]
@ElTacitos That was a bug in the Script Kit app. I believe I fixed it today. Can you test out this pre-release build and let me know if it works as expected?

https://github.com/johnlindquist/kitapp/releases/tag/v1.50.1

**Reply by ElTacitos** [reply]
It is indeed fix thanks a lot mate
